lib/mayaUsd/render/mayaToHydra/renderGlobals.cpp

```diff
@@ -42,98 +44,251 @@ TF_DEFINE_PRIVATE_TOKENS(
     (mtohTextureMemoryPerTexture)
     (mtohColorSelectionHighlight)
     (mtohColorSelectionHighlightColor)
-    (mtohColorSelectionHighlightColorA)
```

Comment:
I'll see if the template makes this guy easier to read, but generally curious if dropping the GfVec4 variant all-together is better ?
There is one Storm selection setting that does rgba, but the rest of Hydra & Shading for USD seems to be discouraging any use of rgba.

@mattyjams has an opinion on this ?

Reply:
Not a particularly well informed one I'm afraid, as I'm not that familiar with the mayaToHydra code, and not really in the loop on the Hydra-side work in this area.

But there has been a trend recently to separate out the handling of color and opacity, for example the change here:
https://github.com/PixarAnimationStudios/USD/commit/332db31e5c0c631e404d5f20c17e8e0d6b30a630

That doesn't extend everywhere yet as you pointed out though, since `UsdImagingGLEngine` and `HdxTaskController` at least so far still expect to be given the selection color as a `GfVec4f`.

So yeah, not sure if that helps at all. :/

Reply:
At least clarifies that supporting GfVec4 needs to stay for a while, and probably won't reduce much complexity having to support version before anyways...

So yeah, thanks!